backend/routes/cryptos.js

```diff
@@ -0,0 +1,204 @@
+var express = require('express');
+var router = express.Router();
+const client = require('../config/clientPg')
+
+function errorQuery(e, res) {
+    res.status(418).json({
+        error: `bdd request failed`,
+        message: `I'm a teapot`
+    })
+    console.error(e.stack)
+}
+
+/* GET cryptos infos
+/cryptos?[cmids={cm}]
+examples:
+    - /cryptos
+    - /cryptos?cmids=BTC
+    - /cryptos?cmids=BTC,ETH,XRP
+*/
+router.get('/cryptos', function (req, res, next) {
+    let query1 = "SELECT * from CRYPTO_LIST",
```

Comment:
```suggestion
    const GET_CRYPTO_LIST = "SELECT * from CRYPTO_LIST",
```
Also, should be a global